cijoe/scripts/xnvme_install.py

```diff
@@ -23,7 +23,11 @@ def main(args, cijoe, step):
     xnvme_source = step.get("with", {}).get("xnvme_source", conf["repository"]["path"])
 
     commands = [
-        "meson install -C builddir",
+        "cd builddir && meson compile",
+        "cd builddir && meson install",
+        "cd builddir && meson --internal uninstall",
```

Comment:
Is this really necessary?

Reply:
In some scenarios yes, in others no, to re-use the same script for multiple scenarios then it is a bit excessive.